test/e2e/e2e_test.go

```diff
@@ -46,7 +69,16 @@ var _ = BeforeSuite(func() {
 	ctx = context.Background()
 })
 
+// runScript runs a script either locally or remotely
+// if the sshKey is provided, it runs the script remotely
 func runScript(script string) (string, error) {
+	if sshKey != "" {
+		return runRemoteScript(script)
+	}
+	return localRunScript(script)
+}
+
+func localRunScript(script string) (string, error) {
```

Comment:
I wonder if I should move everything from this line forward to the `utils.go` file

Reply:
I think we could move all the script specifics to a separate file, but not `utils.go`.

Reply:
Sorry, I should have been more clear. I think we should separate logic according to functionality / domain and name the files accordingly. Using `tools.go` as a catch-all does not achieve this.